.circleci/config.yml

```diff
@@ -1119,6 +730,22 @@ jobs:
                   perform-login: true
             - *delete_gke_jobs
 
+    run_tests:
```

Comment:
I can't find when this job is run.

Reply:
Ah ah, I was using it for my internal testing. Will remove!